scripts/build_podspec.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftNIO open source project
+##
+## Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+set -eu
+
+function usage() {
+  echo "$0 [-u] VERSION"
+  echo
+  echo "OPTIONS:"
+  echo "  -u: Additionally upload the podspec"
+}
+
+upload=false
+while getopts ":u" opt; do
+  case $opt in
+    u)
+      upload=true
+      ;;
+    \?)
+      usage
+      exit 1
+      ;;
+  esac
+done
+shift "$((OPTIND-1))"
+
+if [[ $# -eq 0 ]]; then
+  echo "Must provide target version"
+  exit 1
+fi
+
+VERSION=$1
+NAME="SwiftNIOTransportServices"
+
+here="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+tmpfile=$(mktemp -d /tmp/.build_podspecsXXXXXX)
```

Comment:
that's not a file (maybe in the UNIX sense :P) so maybe `tmpdir`? Also we never delete it

Reply:
Yeah the non-deletion is somewhat intentional, as it lets you view the output before you do anything with it. Open to changing that if you'd prefer.

Reply:
na, its' fine